src/imitation/rewards/reward_nets.py

```diff
@@ -318,6 +302,210 @@ def forward(self, state, action, next_state, done):
         return outputs
 
 
+class NormalizedRewardNet(RewardNetWrapper):
+    """A reward net that normalizes the output of its base net.
+
+    Only requires the implementation of a forward pass (calculating rewards given
```

Comment:
Huh? I think a forward pass is implemented in this class...

Reply:
deleted!